src/pravin/BankDetails.java

```diff
@@ -0,0 +1,65 @@
+package pravin;
+
+public class BankDetails {
+
+	String custName;
+	int totalAmt;
+	int debit = 0;
```

Comment:
Not require to give 0 value to variable, if you are not initialised any variable with any value then it will initialised with it's default value.